wasm/event-stubs/Earthfile

```diff
@@ -0,0 +1,32 @@
+VERSION 0.7
+
+# cspell: words autodrop CFLAGS mexec
+
+# build-c-bindings - generates `wit-bindgen` with C, this should generate `hermes.c`, `hermes.h`, and `hermes_component_type.o`
+build-c-bindings:
+    FROM ../wasi-hermes-component-adapter+builder
+
+    # FIXME: right now we need to remove `--no-object-file` flag to make it runs with our bench, should we add this back?
+    RUN wit-bindgen c --autodrop-borrows yes ../wasi/wit
+
+    SAVE ARTIFACT ./
+
+# build-c-stub - compiles the generated C files into .wasm module
+build-c-stub:
+    FROM ghcr.io/webassembly/wasi-sdk:wasi-sdk-21
+    WORKDIR /repo
+    COPY . .
+    COPY +build-c-bindings/ ./
+
+    RUN $CC hermes.h hermes_component_type.o my-component.c -o out.wasm -mexec-model=reactor $CFLAGS
+
+    SAVE ARTIFACT ./out.wasm
+
+# build-c-component - turns a C .wasm module into a component ready to be used with Hermes
+build-c-component:
```

Comment:
Are you sure that this target is the correct one ?
How you are propagating `out.wasm` file which is generated inside `build-c-stub` ?

Reply:
I missed that. It needs to copy the result from `build-c-stub`, and processes it. Just wonder why the CI didn't catch any errors.